### ISSUE TITLE
'Rename' object not iterable

### ISSUE THREAD

**Issue by seaspatial** [nco/pynco#45]
I am unable to successfully use the nco.ncrename command using pynco 1.0 with nco 4.8.2 on Linux. The error is: 
```
File "/home/arcgis/arcgis/miniconda3/envs/arcpython/lib/python3.6/site-packages/nco/nco.py", line 210, in get cmd.extend(option)
TypeError: 'Rename' object is not iterable
```

Here is the code I am using:
```python
        for name in ['dimension', 'variable']:
            rDict = param['names'][name]
            nco.ncrename(input=nc_file, options=[Rename(name, rDict)])
```

Am I doing something wrong or is there a bug? I have no problems renaming when using nco directly from the terminal.

Thanks!

**Comment by HarshulShuk** [comment]
I have also been having this same issue for quite some time. I run into a similar problem when using Rename

**Comment by chuaxr** [comment]
I had some trouble too, but found that this worked for me on both nco 4.5.3 and 4.6.2:

```
from nco import Nco
from nco.custom import Rename
nco = Nco()
rename_dict = dict({'old_name':'new_name'})
nco.ncrename(
    input="in.nc",
    output="out.nc",
    options=[Rename("v",rename_dict).prn_option()]
)
```

**Comment by hmb1** [comment]
@HarshulShuk 
@seaspatial 

HI Guys,
  am aware of this problem and have patch that fixes this so that teher is no need to add the method "prn_option()" to the "Rename" onject

...Henry

**Comment by ferrighi** [comment]
Hi @hmb1 , 
 do you have this patch available? or is it still work in progress?

**Comment by hmb1** [comment]
@ferrighi 
I made a pull request back in january 
Waiting on @jhamman to review it

**Comment by peterjsadowski** [comment]
I just installed pynco via anaconda and had the same problem, so chuaxr's solution was helpful.

**Comment by chrisdwells** [comment]
Hi, 

Just to note, I was having the same issue with Limit for ncks, and solved it with the same solution, as in (with dimension "year"):

`nco.ncks(input="in.nc", output="out.nc", options=[Limit("year", 0, 1).prn_option()])
`

so thanks for the tip.

**Comment by czender** [comment]
@hmb1 multiple people have shown this issue (that your January 2020 patch was supposed to have fixed) still exists. It affects at least ncks and ncrename (and possibly other) operators. This is unfinished business. Please submit a new patch that resolves this issue. Thanks!

**Comment by lukelbd** [comment]
Weirdly, it seems this is fixed for version 1.0.0 on macOS, but not on linux. I think there was a mistake when preparing the 1.0.0 release? This is the output of `pip show nco` on both systems:

```sh
Name: nco
Version: 1.0.0
Summary: python bindings to NCO
Home-page: https://github.com/nco/pynco
Author: Joe Hamman
Author-email: jhamman@ucar.edu
```

A workaround would be to install from the git repository using `pip install git+https://github.com/nco/pynco.git`.

**Comment by mx-moth** [comment]
Version 1.1.0 has been released which fixes this issue

**Comment by chuaxr** [comment]
@mx-moth thanks, where is this available? Right now the release on github is still marked at 1.0.0, and I don't see anything earlier in pip/conda.

**Comment by mx-moth** [comment]
For some reason the conda-forge automated release had not succeeded. I fixed that. 1.1.0 has been released on conda-forge now ([link](https://anaconda.org/conda-forge/pynco)).

1.1.0 has also been published on PyPI, but under the `nco` name ([link](https://pypi.org/project/nco/)). The `pynco` name on PyPI has not been updated since 2015. I will make an issue to sort out the `nco`/`pynco` confusion on PyPI
